tests/cassettes/TestAddressCreate.yaml

```diff
@@ -6,27 +6,29 @@ interactions:
       Francisco","state":"CA","zip":"94107","name":"Jack Sparrow","company":"EasyPost","phone":"5555555555"}}'
     form: {}
     headers:
+      Authorization:
+      - REDACTED
       Content-Type:
       - application/json
       User-Agent:
-      - EasyPost/v2 GoClient/2.0.1 Go/go1.17.5 OS/darwin
+      - REDACTED
     url: https://api.easypost.com/v2/addresses
     method: POST
   response:
-    body: '{"id":"adr_5408ef399b2511ec9bc7ac1f6b0a0d1e","object":"Address","created_at":"2022-03-03T19:08:36+00:00","updated_at":"2022-03-03T19:08:36+00:00","name":"Jack
-      Sparrow","company":"EasyPost","street1":"388 Townsend St","street2":"Apt 20","city":"San
-      Francisco","state":"CA","zip":"94107","country":"US","phone":"5555555555","email":null,"mode":"test","carrier_facility":null,"residential":null,"federal_tax_id":null,"state_tax_id":null,"verifications":{}}'
+    body: '{"carrier_facility":null,"city":"San Francisco","company":"EasyPost","country":"US","created_at":"2022-05-24T22:28:07+00:00","email":"REDACTED","federal_tax_id":null,"id":"adr_c9060649dbb011ec8b29ac1f6bc72124","mode":"test","name":"Jack
```

Comment:
Hmm... because we are converting to a string and back again, the ordering is getting all messed up. In a perfect world, the keys would stay ordered in the order they were defined in the struct (old change on the left) vs now they appear to be ordered alphabetically. Is there a way we can preserve the old ordering?

Reply:
`json.Unmarshal` converts the json string to a map with random order every time because the map is unordered in Golang, and `json.Marshal` convert map to a json string in alphabetical order

Yes, it's possible if we implement the `json.Marshaler` interface on a custom type. Code has been added and re-recorded cassettes.